lib/chef/resource_reporter.rb

```diff
@@ -271,6 +276,7 @@ def prepare_run_data
       run_data["data"] = {}
       run_data["start_time"] = start_time.to_s
       run_data["end_time"] = end_time.to_s
+      run_data["expanded_run_list"] = Chef::JSONCompat.to_json(@expanded_run_list)
```

Comment:
Kinda weird that this is double-JSON-encoded but it's consistent with the other stuff so it's okay I guess.